### PR TITLE
[FLINK-10563] Expose shaded Presto S3 filesystem under "s3p" scheme

### PR DESCRIPTION
**Pull request by aljoscha**

### CODE REVIEW

**Comment by zentol** [comment]
the shading check must be updated

**Comment by aljoscha** [comment]
fixed

File: flink-filesystems/flink-s3-fs-presto/src/main/java/org/apache/flink/fs/s3presto/S3PFileSystemFactory.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.fs.s3presto;
+
+/**
+ * Simple factory for the S3 file system, registered for the <tt>s3a://</tt> scheme.
+ */
```
Comment (kl0u) [inline flink-filesystems/flink-s3-fs-presto/src/main/java/org/apache/flink/fs/s3presto/S3PFileSystemFactory.java]: Update the javadoc.

**Comment by aljoscha** [comment]
merged

**Pull request closed without merging**